piet-cairo/Cargo.toml

```diff
@@ -1,17 +1,24 @@
 [package]
 name = "piet-cairo"
-version = "0.7.0"
+version.workspace = true
 authors = ["Raph Levien <raph.levien@gmail.com>"]
```

Comment:
We should probably move to the AUTHORS file, I presume that would be a follow-up?

Reply:
Might as well do it here as it's quite straightforward. Added the commit.